### ISSUE TITLE
Install script resets mouse speed on Windows 7

### ISSUE THREAD

**Issue by GlowingCrystallineEntity** [RaiMan/SikuliX-2014#169]
The mouse speed control, accessed through Control Panel -> Personalization -> Change mouse pointers -> Pointer Options, is reset to "medium" by the install script.  Seems to be near the end, maybe during the validation process.  (I always set it to "Fast.")  (Not the biggest deal, but annoying, especially when re-installing nightly builds.)

(p.s. -- thanks for building Sikuli.  It's very cool. :) )

![image](https://cloud.githubusercontent.com/assets/11333261/10118801/270f50c4-6450-11e5-8836-1ae66be1b5c1.png)

**Comment by GlowingCrystallineEntity** [comment]
Actually, this happens (also?) when the IDE starts.

**Comment by RaiMan** [comment]
I am sure, that SikuliX does not actively do that.
Must have something to do with the used Java Robot class.
Supposing it is Win 7-64: what Java do you use?

**Comment by RaiMan** [comment]
Of course I will check that ;-)

**Comment by GlowingCrystallineEntity** [comment]
Yes, it's Win7-64.  I'm using JDK 1.8.0_20.  (A bit old, granted...)

Your script reports:

Java(TM) SE Runtime Environment (build 1.8.0_20-b26)
Java HotSpot(TM) 64-Bit Server VM (build 25.20-b23, mixed mode)


Of note, it only happens irregularly, though often enough.  Anyway, thanks for checking.

**Comment by GlowingCrystallineEntity** [comment]
More info:  I (accidentally) double-clicked sikulixapi.jar in the Sikuli install folder.  It brought up some JavaScript dialog.  I clicked "cancel" on that, and when it closed, it reset my mouse as above.

**Comment by RaiMan** [comment]
Confirmed.
I can reproduce this on Windows 7 and even 10.
The problem is due to some buggy interaction between Java Robot and Windows (not sure who is responsible ;-) 
I could not find any hint, that this is reported as abug neither in Java nor in Windows.
Nevertheless I accept this as a bug.
A possible solution might be to reset the mouse speed to the value before using Java Robot the first time, after a script is run or latest when the JVM running SikuliX shuts down.  

Since this might take a while, you have to live with it currently.

Interesting links:
http://stackoverflow.com/questions/9986423/java-awt-robot-changes-windows-mouse-speed
http://manjeetdahiya.com/2011/06/08/control-mouse-speed-from-command-line/

**Comment by GlowingCrystallineEntity** [comment]
Ah, thanks for confirming this, and thanks for finding that command line utility -- that looks useful.
